src/Util/ColumnParser.php

```diff
@@ -191,7 +190,6 @@ public function getType($field, $type)
         $collection = new Collection($reflector->getConstants());
 
         $validTypes = $collection->filter(function ($value, $constant) {
-            $value;
```

Comment:
Mine does too complain about an unused variable, but it also complained about the previous state.
It gave me a "Expression result unused".
So what do we do ?